machine_learning/xgboostclassifier.py

```diff
@@ -0,0 +1,75 @@
+# XGBoost Classifier Example
+import numpy as np
+from matplotlib import pyplot as plt
+from sklearn.datasets import load_iris
+from sklearn.metrics import plot_confusion_matrix
+from sklearn.model_selection import train_test_split
+from xgboost import XGBClassifier
+
+
+def data_handling(data: dict) -> tuple:
+    # Split dataset into train and test data
+    # data is features
+    """
+    >>> data_handling(({'data':'[5.1, 3.5, 1.4, 0.2]','target':([0])}))
+    ('[5.1, 3.5, 1.4, 0.2]', [0])
+    >>> data_handling({'data':'[4.9, 3. , 1.4, 0.2],[4.7, 3.2, 1.3, 0.2]','target':([0, 0])})
```

Comment:
machine_learning/xgboostclassifier.py:16:89: E501 line too long (93 > 88 characters)
```suggestion
    >>> data_handling(
    ...     {'data':'[4.9, 3. , 1.4, 0.2],[4.7, 3.2, 1.3, 0.2]','target':([0, 0])}
    ... )
```